### DISCUSSION TITLE
LILYGO® TTGO LoRa32 V2.1_1.6 Version meshtastic flashing

### DISCUSSION THREAD

**Discussion by sod121a** [meshtastic/firmware#3313] [Q&A] [answered]
So this device is not supported without manually entering code.  I think its a little above me. Any suggestions on where to start or just am i SOL? Thank you

**Comment by GUVWAF** [comment, marked as answer]
It is listed at the [Web Flasher](https://flasher.meshtastic.org/) as "T-LoRa V2.1-1.6".

**Reply by sod121a** [reply]
Thank you.

**Reply by sod121a** [reply]
I was able to get it on the device but now I get an error three code. This happen you you.

**Reply by GUVWAF** [reply]
Critical Error Code 3 means no radio could be found. Do you have a picture or product link to the device you have? To make sure you flashed the correct firmware.

**Reply by sod121a** [reply]
| LILYGO® TTGO LoRa32 V2.1_1.6 Version 915Mhz ESP32 LoRa OLED 0.96 Inch SD Card Bluetooth WIFI Wireless Module ESP-32  SMA
https://a.aliexpress.com/_mqXvWmW 

Is the one I have

**Reply by GUVWAF** [reply]
For that one indeed you need "T-LoRa V2.1-1.6". Are you sure you didn't accidentally flash "T-LoRa V2.1-1.**8**"?

**Reply by sod121a** [reply]
Thanks so much I got it working now.
